### DISCUSSION TITLE
[A-L] (2023/24) Foglio 17 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#384] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/5b974cbb-a494-435b-9690-f46323526e40)

**Comment by Elia-Belli** [comment, marked as answer]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/d8209272-3fc0-446d-82b9-7c9d4a1ef1de)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/cfc27ee0-b0a2-49e0-8ea4-a2f803fd9282)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/43184888-251f-46ae-bd7e-abb936568b24)

Spero si legga (e sia corretto)

**Reply by FrascaHoryzont** [reply]
Scusa bro, ti volevo chiedere spiegazione riguardo a questa parte dell'esercizio, poiché mi è poco chiaro perché la retta associata sia R(1,-1,0) al quale non mi torna il valore del -1 in x_2..
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/100992294/1d0d3a70-4510-4981-9f58-7877f27f093a)

**Reply by Frisbon** [reply]
Nota che una base non é unica, se la mia la moltiplichi per -1 viene R(-1,1,0) 
Comunque ho rifatto il calcolo, tié.

![image_2024-01-27_16-11-33](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/d609be56-d38d-4053-b362-d9ef0b093d30)

(semplifico e divido la matrice per 2, sposto la seconda riga sulla prima, i pivot sono variabili dipendenti, le libere fanno lo span)

**Reply by FrascaHoryzont** [reply]
> Nota che una base non é unica, se la mia la moltiplichi per -1 viene R(-1,1,0) 
> 
> Comunque ho rifatto il calcolo, tié.
> 
> 
> 
> ![image_2024-01-27_16-11-33](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/d609be56-d38d-4053-b362-d9ef0b093d30)
> 
> 
> 
> (semplifico e divido la matrice per 2, sposto la seconda riga sulla prima, i pivot sono variabili dipendenti, le libere fanno lo span)

Perfetto grazie mille 🫡
